.github/ISSUE_TEMPLATE/01-github-request---onboarding.yml

```diff
@@ -49,7 +54,7 @@ body:
     id: requests
     attributes:
       label: Special Requests
-      description: For now, all new 2U/edX employees will be granted access to `push-pull-all`. If these users need to be added to any specific teams or given any elevated privileges, note them here. 
+      description: For now, all new 2U/edX employees will be granted access to `2u-edx-legacy`. If these users need to be added to any specific Github teams or given any elevated privileges, note them here.
```

Comment:
I thought we weren't adding new people to 2u-edx-legacy?  I know there are newer engineers who are not in the team.

Reply:
Hmm, maybe that was just a hiccup?  It looks like even newer people are in the team.

Reply:
As I understand it, we're still supposed to add people to it if requested.  What's fuzzy to me is if it's _not_ requested.

Reply:
I'm guessing we can deal with the wording here separately, so going ahead and merging the PR.